Trakt/Web/trakt.js

```diff
@@ -0,0 +1,207 @@
+/*Array.prototype.remove = function () {
+    let what;
+    const a = arguments;
+    let L = a.length;
+    let ax;
+    while (L && this.length) {
+        what = a[--L];
+        while ((ax = this.indexOf(what)) !== -1) {
+            this.splice(ax, 1);
+        }
+    }
+    return this;
+};*/
+
```

Comment:
Was this purposefully kept in?

Reply:
oh missing

Reply:
@crobibero done https://github.com/jellyfin/jellyfin-plugin-trakt/pull/78/commits/d783b43bcdf0b3b12ed4ee57078d9ac8cada2564